specification/synapse/resource-manager/Microsoft.Synapse/preview/2020-04-01-preview/workspace.json

```diff
@@ -0,0 +1,1394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-04-01-preview",
```

Comment:
when adding a new api-version please make the first commit the same as the content for the previous api-version and then the diff as the next commit. That makes the review much faster.

Reply:
We are going with same API-version, not updating the API version.